### ISSUE TITLE
Make an easy way to recover a webhook

### ISSUE THREAD

**Issue by diego-lipinski-de-castro** [osiset/laravel-shopify#936]
Happened to me that one of my webhooks was removed duo to excessive error 500.

Shopify says:

"
In the event that your app goes offline for an extended period of time, you can recover your webhook by re-registering your webhook and importing the missing data.

To re-register a webhook, consult the app's code that initially registered the webhook. You can add a check that fetches all the existing webhooks and registers only the ones that you need.

To import the missing data, you can fetch data from the outage period and feed it into your webhook processing code.

".

So I think would be nice to have an easy way to reinstall (or recover) webhooks.

**Comment by enmaboya** [comment]
Just make a rest or graphql query, it's easy.
https://shopify.dev/apps/webhooks#1-subscribe-to-a-webhook-topic

**Comment by diego-lipinski-de-castro** [comment]
I dont know exactly how I should do that inside the project. Does this needs to be called only once or once for each shop?

**Comment by Kyon147** [comment]
Webhooks are transient they only exists the moment they have been fired. 

Shopify is basically saying you can re-register missing webhooks by checking what webhooks are missing but this happens on authenticate anyway.

Missing data, as @enmaboya mentioned you would need to use the API to capture the missing data from the period you are missing manually or by setting up a job.

Closing this as it is not a bug or issue related to the package.
